quodlibet/quodlibet/browsers/playlists/main.py

```diff
@@ -615,7 +615,8 @@ def reordered(self, songs):
             playlist = model[iter][0]
             playlist[:] = songs
         elif songs:
-            playlist = FileBackedPlaylist.from_songs(PLAYLISTS, songs)
+            playlist = FileBackedPlaylist.from_songs(PLAYLISTS, songs,
+                                                     self.__class__.library)
```

Comment:
I did not realise that there was a `self.library` attribute. I guess I assumed that there would not be one since `library` was passed as a parameter to several methods, and forgot to actually check. I fixed this in the last commit I pushed.